easybuild/easyconfigs/g/GLIMMER/GLIMMER-3.02b-goolf-1.4.10.eb

```diff
@@ -20,13 +21,27 @@ toolchain = {'name': 'goolf', 'version': '1.4.10'}
 source_urls = ['http://www.cbcb.umd.edu/software/glimmer']
 sources = ['%%(namelower)s%s.tar.gz' % ''.join(version.split('.'))]
 
+dependencies = [('ELPH', '1.0.1')]
+
 buildopts = '-C ./src'
 
 files_to_copy = ["bin", "docs", "glim302notes.pdf", "lib", "LICENSE", "sample-run", "scripts"]
 
+# add scripts dir to PATH
+modextrapaths = {
+	'PATH': 'scripts'
+}
+
 sanity_check_paths = {
     'files': ["bin/glimmer3", "glim302notes.pdf", "LICENSE"],
     'dirs': ["docs", "lib", "sample-run", "scripts"]
 }
 
+# fix hardcoded path to scripts folder
+postinstallcmds = ["sed -i -e 's|/fs/szgenefinding/Glimmer3/scripts|%(installdir)s/scripts|' %(installdir)s/scripts/*"]
+# fix hardcoded path to bin folder
+postinstallcmds += ["sed -i -e 's|/fs/szgenefinding/Glimmer3/bin|%(installdir)s/bin|' %(installdir)s/scripts/*"]
+# fix hardcoded path to elph binary
+postinstallcmds += ["sed -i -e 's|/nfshomes/adelcher/bin/elph|${EBROOTELPH}/bin/elph|' %(installdir)s/scripts/*"]
```

Comment:
oh boy :)

there's no way to do this via a patch file up front? I'm not a big fan of relying on `sed` for this, and post-install patching if this can be changed prior to installation